teachopencadd/talktorials/T016_protein_ligand_interactions/talktorial.ipynb

```diff
@@ -4,201 +4,186 @@
    "cell_type": "markdown",
```

Comment:
<p>I suggest use:</p><pre class="ql-syntax" spellcheck="false">create_df_from_binding_site(list(interactions_by_site.keys())[0],
&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;interaction_type="hydrophobic")
</pre><p>to make the code more readable.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/75/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>